src/components/select/_select.scss

```diff
@@ -121,14 +121,22 @@
   }
 
   .#{$prefix}--select--inline {
-    display: flex;
-    flex-direction: row;
-    align-items: center;
+    display: grid;
+    grid-template-columns: auto auto;
+    display: -ms-grid;
+
+    // Targets IE10+ browsers: Display grid auto not supported
+    @media all and (-ms-high-contrast: none), (-ms-high-contrast: active) {
+      display: flex;
+      flex-direction: row;
+      align-items: center;
+    }
 
     .#{$prefix}--label {
       white-space: nowrap;
       margin: 0 $spacing-xs 0 0;
       font-weight: 400;
+      align-self: center;
```

Comment:
Wanted to see if Safari is OK with the change here.

Reply:
Just double checked and this is displaying correctly in Safari.